### PR TITLE
feat(): override md-date-time style

### PR DESCRIPTION
**Pull request by FlorentMarima**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -20,11 +20,14 @@
     "bootstrap-sass-official": "~3.3.1",
     "font-awesome": "~4.3.0",
     "bootstrap-material-design": "~0.3.0",
-    "mdi": "~1.1.34"
+    "mdi": "~1.1.34",
+    "md-date-time": "~0.0.14"
```
Comment (adlenafane) [inline bower.json]: Is it really needed?
@birdy- Should this be migrated to `package.json`?

**Comment by adlenafane** [comment]
LGTM

File: bower.json (hunk 1 above)
Comment (birdy-) [inline bower.json]: I'll clean this bower.json and remove all dependencies in it. So it doesn't matter atm.

Comment (birdy-) [inline bower.json]: In fact it's good to keep dependencies in this as reminder. When I'll remove this file, I'll check that everything is fine with these dependencies.

**Pull request merged**